pkg/mkimage/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM linuxkit/alpine:07f7d136e427dc68154cd5edbb2b9576f9ac5213 AS mirror
+FROM linuxkit/alpine:d307c8a386fa3f32cddda9409b9687e191cdd6f1 AS mirror
```

Comment:
This package is in turn used as `FROM` in `mkimag-{gcp,vhd,vmdk,dynamic-vhd}` but those aren't updated here.

Reply:
They seem to use `linuxkit/guestfs` which in turns is based on `debian:stretch`

Reply:
Oh yes, my mistake. I somehow misread `pkg/mkimage` as `tools/guestfs`! Sorry for the noise.